test/integration/test.js

```diff
@@ -48,14 +60,21 @@ runtimes.forEach(runtime => {
 
     before(async function() {
       this.timeout(0);
-      process.env.RUNTIME = runtime
-      await run('deploy');
+      process.env.RUNTIME = runtime;
+      const { subprocess, output } = await runSpawned('offline');
+      endpoints = getEndpoints(output);
+      this.subprocess = subprocess;
     });
 
-    before(async function() {
-      this.timeout(10000);
-      const info = await run('info');
-      endpoints = await getEndpoints(info.split('\r?\n'));
+    after(async function() {
+      let done = false;
+      this.subprocess.on('close', () => {
+        done = true;
+      })      
+      kill(this.subprocess.pid);
```

Comment:
Tried doing this with `this.subprocess.signal('SIGKILL')`, etc but no luck. [tree-kill](https://www.npmjs.com/package/tree-kill) OTOH works fine!